interface-spec.md

```diff
@@ -4,10 +4,10 @@
 **Purpose:** This document proposes a uniform interface to represent RDF data in low-level JavaScript libraries.
 
 ## Design elements and principles
-- We define data interfaces to represent **triples**, **quads**, **IRIs**, **blank nodes**, **literals** and **variables**.
+- We define data interfaces to represent **triples**, **quads**, **named nodes**, **blank nodes**, **literals** and **variables**.
```

Comment:
Discussed in the 2015-11-27 meeting: We like to follow the RDF abstract syntax and keep "IRI" as a term.